draft-ietf-keytrans-protocol.md

```diff
@@ -1805,6 +1772,91 @@ each label are not provided, given that these can be cached from the
 original Search or Update query.
 
 
+# Third Parties
+
+## Management
+
+With the Third-Party Management deployment mode, a third party is responsible
+for the majority of the work of storing and operating the Transparency Log,
+while the Service Operator serves mainly to enforce access control and
+authenticate the addition of new entries. All user queries specified in
+{{user-operations}} are initially sent by users directly to the Service Operator
+to be forwarded to the Third-Party Manager if they pass access control.
+
+While other operations are forwarded by the Service Operator unchanged,
+`UpdateRequest` structures are forwarded to the Third-Party Manager with the
+Service Operator's signature attached:
+
+~~~ tls-presentation
+struct {
+  UpdateRequest request;
+  opaque signature<0..2^16-1>;
+} ManagerUpdateRequest;
+~~~
+
+The signature is computed as described in {{update-format}}. The Service
+Operator MUST maintain its own records (independent of the Third-Party Manager)
+for the greatest version of each label for the purpose of producing this
+signature.
+
+## Auditing
+
+With the Third-party Auditing deployment mode, the Service Operator obtains
+signatures from a lightweight Third-Party Auditor attesting to the fact that the
+Service Operator is constructing the tree correctly. These signatures are
+provided to users along with the responses to their queries.
+
+Each entry added to the log tree has a corresponding `AuditorUpdate` structure
```

Comment:
You say that each entry "has a corresponding `AuditorUpdate` structure. I find it unclear what that means. Will it be added to the log tree itself? I presume not as it is sufficient to show this to single auditors and need not be committed to. Maybe "`AuditorUpdate` structures correspond to log tree entries and ...". Or say that auditors can request them.